.gitignore

```diff
@@ -16,3 +16,5 @@ tmp/
 # Mac OSX Finder files.
 **/.DS_Store
 .DS_Store
+
+yarn.lock
```

Comment:
There's never a case where this file is not wanted. If you use npm, you won't have this file. If you use yarn, you want to commit it to git.